src/System.Private.Xml/src/System/Xml/Serialization/Compilation.cs

```diff
@@ -244,11 +254,31 @@ internal static Assembly LoadGeneratedAssembly(Type type, string defaultNamespac
             contract = (XmlSerializerImplementation)Activator.CreateInstance(contractType);
             if (contract.CanSerialize(type))
                 return serializer;
-
+            Log(SR.Format(SR.XmlSerializerExpiredDetails, serializerName, type.FullName), EventLogEntryType.Error);
             return null;
         }
 
+        private static void Log(string message, EventLogEntryType type)
+        {
+            new EventLogPermission(PermissionState.Unrestricted).Assert();
+            EventLog.WriteEntry("XmlSerializer", message, type);
```

Comment:
EventLog isn't in .NET Standard but is in .NET Standard + Platform Extensions so I don't believe this can be used as this code is part of .NET Standard. We might want to consider replacing this with an EventSource instead. I would remove this logging and open an issue mentioning that we don't have this information logged and to consider adding an EventSource for this.